components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/ConfigurationLoader.java

```
@@ -79,10 +105,27 @@
                 if (handlerElem.getAttribute(CLASS_Q) != null) {
```

We need to throw an exception if the class attribute itself is null